Modules/Scripted/WebServer/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(MODULE_NAME WebServer)
 set(MODULE_PYTHON_SCRIPTS
   ${MODULE_NAME}.py
   ${MODULE_NAME}Lib/__init__
+  ${MODULE_NAME}Lib/base.py
```

Comment:
Looks like you still need to change the name here?

Reply:
Good catch!